connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerConnectorTest.java

```diff
@@ -60,15 +64,15 @@ public class WorkerConnectorTest extends EasyMockSupport {
     public ConnectorConfig connectorConfig;
     public MockConnectMetrics metrics;
 
-    @Mock Plugins plugins;
-    @Mock SourceConnector sourceConnector;
-    @Mock SinkConnector sinkConnector;
-    @Mock Connector connector;
-    @Mock CloseableConnectorContext ctx;
-    @Mock ConnectorStatus.Listener listener;
-    @Mock CloseableOffsetStorageReader offsetStorageReader;
-    @Mock ConnectorOffsetBackingStore offsetStore;
-    @Mock ClassLoader classLoader;
+    private final Plugins plugins = mock(Plugins.class);
+    private final SourceConnector sourceConnector = mock(SourceConnector.class);
+    private final SinkConnector sinkConnector = mock(SinkConnector.class);
+    private final CloseableConnectorContext ctx = mock(CloseableConnectorContext.class);
+    private final ConnectorStatus.Listener listener = mock(ConnectorStatus.Listener.class);
+    private final CloseableOffsetStorageReader offsetStorageReader = mock(CloseableOffsetStorageReader.class);
+    private final ConnectorOffsetBackingStore offsetStore = mock(ConnectorOffsetBackingStore.class);
+    private final ClassLoader classLoader = mock(ClassLoader.class);
+    private Connector connector;
```

Comment:
Is there a reason that this field is a member? Perhaps we could use a local variable in each of the test for this.

Reply:
I think it helps with readability a bit. We always need a `Connector` for our tests. The reason we can't use the `@Mock` annotation for it is just that we always use either the (mocked) `SourceConnector` or `SinkConnector`.

Also worth noting that if https://github.com/apache/kafka/pull/12307 lands, a class-level `Connector` would be easier to work with.

Reply:
ok, I don't have strong opinion on this one. We can keep it as it is.